params/version.go

```diff
@@ -24,7 +24,7 @@ import (
 const (
 	VersionMajor = 5         // Major version component of the current release
 	VersionMinor = 2         // Minor version component of the current release
-	VersionPatch = 0         // Patch version component of the current release
+	VersionPatch = 1         // Patch version component of the current release
```

Comment:
Is this the PR that enables SHA-256 in CCC? Do we need to deploy it before the upgrade (Mon) or after it? Is it backward compatible (i.e. it's fine if node operators don't upgrade)?

Reply:
this PR should be done after the upgrade. as it's based on 0.10 instead of 0.9
that said, for the safety we'd better don't merge it now, in case we have things to fix during the upgrade.

@colinlyguo see the discussion here https://github.com/scroll-tech/go-ethereum/pull/693#issuecomment-2041766451

Reply:
Also see discussion [here](https://scrollco.slack.com/archives/C06H97VT94M/p1712806306246009?thread_ts=1712805600.707909&cid=C06H97VT94M).

Reply:
makes sense.